src/zcbor_any_skip_fixed.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __ZCBOR_ANY_SKIP_FIXED_H__
+#define __ZCBOR_ANY_SKIP_FIXED_H__
+
+#if defined(ZCBOR_VARIANCE) && ZCBOR_VERSION_MAJOR == 0 && ZCBOR_VERSION_MINOR < 7
```

Comment:
I opted to introduce the `ZCBOR_VARIANCE` symbol here rather than calculating the zcbor version in each of the ports because we are providing the `zcbor` library for them, which we know to be pinned to `0.7`.